shell/platform/darwin/macos/framework/Headers/FLETexture.h

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
@chinmaygarde 
I found in the macos test that CVOpenGLTextureCacheCreateTextureFromImage creates a GL_TEXTURE_RECTANGLE object instead of GL_TEXTURE_2D, which causes the texture to not be properly scaled to fill the flutter widget when rendering, so I used width/height in texture->copyPixelBuffer when the plugin receives the request, it scales the texture to the corresponding size according to the widget size provided by the engine, and then becomes normal when rendering.
I'm not sure to add width/height to ios/framework/Headers/FlutterTexture.h, or continue to use FLETexture.h, or should report a new issue in flutter engine.

Reply:
You should be able to specify that the texture target is GL_TEXTURE_RECTANGLE by specifying the [target property of FlutterOpenGLTexture](https://github.com/flutter/engine/blob/91b71070f55ffc38c432babc7d9bb68ac2720d8d/shell/platform/embedder/embedder.h#L194). All the copying ought to be unnecessarily inefficient.